addons/hr_recruitment/models/hr_applicant.py

```diff
@@ -611,6 +611,12 @@ def _compute_date_closed(self):
             if not applicant.stage_id.hired_stage:
                 applicant.date_closed = False
 
+    def copy(self, default=None):
```

Comment:
Copy is supposed to be called in batch, hence this override is breaking the intended behavior.

Override ``copy_data`` instead:
```py
    def copy_data(self, default=None):
        vals_list = super().copy_data(default=default)
        return [dict(vals, name=self.env._("%s (copy)", applicant.name)) for applicant, vals in zip(self, vals_list)]
```